opencti-platform/opencti-front/src/private/components/dataEntities/data/DataSourceCreation.js

```diff
@@ -0,0 +1,370 @@
+import StixDomainObjectHeader from '../../common/stix_domain_objects/StixDomainObjectHeader';
+import RiskCreationOverview from '../../riskAssessment/risks/RiskCreationOverview';
+import CyioCoreObjectLatestHistory from '../../common/stix_core_objects/CyioCoreObjectLatestHistory';
+import CyioCoreObjectOrCyioCoreRelationshipNotes from '../../analysis/notes/CyioCoreObjectOrCyioCoreRelationshipNotes';
+import CyioCoreObjectAssetCreationExternalReferences from '../../analysis/external_references/CyioCoreObjectAssetCreationExternalReferences';
+import Loader from '../../../../components/Loader';
+import RiskCreationDetails from '../../riskAssessment/risks/RiskCreationDetails';
+
+const styles = (theme) => ({
+  container: {
+    marginBottom: 0,
+  },
+  header: {
+    margin: '-25px -24px 20px -24px',
+    padding: '23px 24px 24px 24px',
+    height: '64px',
+    backgroundColor: theme.palette.background.paper,
+  },
+  gridContainer: {
+    marginBottom: 20,
+  },
+  iconButton: {
+    float: 'left',
+    minWidth: '0px',
+    marginRight: 15,
+    padding: '8px 16px 8px 8px',
+  },
+  title: {
+    float: 'left',
+    textTransform: 'uppercase',
+  },
+  rightContainer: {
+    float: 'right',
+    marginTop: '-10px',
+  },
+  dialogActions: {
+    justifyContent: 'flex-start',
+    padding: '10px 0 20px 22px',
+  },
+  editButton: {
+    position: 'fixed',
+    bottom: 30,
+    right: 30,
+  },
+  drawerPaper: {
+    minHeight: '100vh',
+    width: '50%',
+    position: 'fixed',
+    overflow: 'auto',
+    backgroundColor: theme.palette.navAlt.background,
+    transition: theme.transitions.create('width', {
+      easing: theme.transitions.easing.sharp,
+      duration: theme.transitions.duration.enteringScreen,
+    }),
+    padding: 0,
+  },
+});
+
+const dataSourceCreationMutation = graphql`
+  mutation DataSourceCreationMutation($input: RiskAddInput) {
+    createRisk (input: $input) {
+      id
+      # ...RiskCard_node
+      # ...RiskDetails_risk
+      # operational_status
+      # serial_number
+      # release_date
+      # description
+      # version
+      # name
+    }
+  }
+`;
+
+const riskValidation = (t) => Yup.object().shape({
+  name: Yup.string().required(t('This field is required')),
+  // asset_type: Yup.array().required(t('This field is required')),
+  // implementation_point: Yup.string().required(t('This field is required')),
+  // operational_status: Yup.string().required(t('This field is required')),
+  // first_seen: Yup.date()
+  //   .nullable()
+  //   .typeError(t('The value must be a date (YYYY-MM-DD)')),
+  // last_seen: Yup.date()
+  //   .nullable()
+  //   .typeError(t('The value must be a date (YYYY-MM-DD)')),
+  // sophistication: Yup.string().nullable(),
+  // resource_level: Yup.string().nullable(),
+  // primary_motivation: Yup.string().nullable(),
+  // secondary_motivations: Yup.array().nullable(),
+  // personal_motivations: Yup.array().nullable(),
+  // goals: Yup.string().nullable(),
+});
```

Comment:
@sachin755 regarding the large chunks of commented out code, if there is no intent for them to be used as of now, please remove them to cleanup the code. I will use this as an overall request for all commented code on this PR.